heartbeat/ocf.py

```diff
@@ -248,16 +248,16 @@ def __str__(self):
 		return self.to_xml()
 
 	def to_xml(self):
-		def opt(s, name, var):
+		def opt(s, name, var, time=True):
 			if var is not None:
 				if type(var) == int:
-					var = "{}s".format(var)
+					var = "{}{}".format(var, "s" if time else "")
```

Comment:
Maybe we could just do `if name in ("timeout", "interval")` instead? Seems less hacky.

Reply:
yeah, actually that is probably clearer... I'll change it.